src/main/resources/templates/user.html

```diff
@@ -7,30 +7,57 @@
             border-collapse: collapse;
         }
     </style>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
     <title>Users</title>
 </head>
 <body>
-    <form action="#" th:action="@{/user}" th:object="${users}" method="post">
-        <p>SignUp <a href="/register">here</a></p>
-        <table style="width:30%">
-            <tr>
-                <th>ID</th>
-                <th>Firstname</th>
-                <th>Lastname</th>
-                <th>Username</th>
-                <th>Email</th>
-                <th>Usertype</th>
-            </tr>
-            <tr th:each="item : ${users}">
-                <td><p th:text="${item.getId()}"></p></td>
-                <td><p th:text="${item.getFirstName()}"></p></td>
-                <td><p th:text="${item.getLastName()}"></p></td>
-                <td><p th:text="${item.getUserName()}"></p></td>
-                <td><p th:text="${item.getEmail()}"></p></td>
-                <td><p th:text="${item.getUserType()}"></p></td>
-            </tr>
-        </table>
+    <div id="nav-placeholder"></div><br><br>
+    <form action="#" th:action="@{/user}" th:object="${users}" class="container d-flex flex-column flex-md-col justify-content-between col-sm-12 col-md-6 col-lg-4">
+        <div class="mx-auto" style="height: auto">
+            <table style="width:30%">
```

Comment:
why is the width only 30%?
if possible, table styling should be done with bootstrap classes: https://getbootstrap.com/docs/4.6/content/tables/